remoteit/desktop
# DESK-805 registration timing and updated licensing messages.

### Description



Fix for delayed cloud response to device registration
Settings page device link showing wrong name
Licensing messaging update
CLI update to version 1.6.14

### Test plan



1. Sign in
2. Go to Japan (I can't reproduce here)
3. Click register 
4. You should see the registration finish

### Ticket



<https://remoteit.atlassian.net/browse/DESK-805>
